pom.xml

```diff
@@ -203,6 +459,15 @@
             <verbose>true</verbose>
             <complianceLevel>${jdk.version}</complianceLevel>
             <showWeaveInfo>true</showWeaveInfo>
+            <Xlint>ignore</Xlint>
+            <!-- IMPORTANT -->
```

Comment:
Yeah had some error? can you add what went wrong and why Important please?

Reply:
This comment is for to work Lombok with aspectj